### PR TITLE
Fix KinD typos in en docs

### PR DESCRIPTION
**Pull request by yanggangtony**
Fix ```KinD``` typos in en docs.
There are somes show Kind , and somes show KinD.
Review the website of kind [kind quick start](https://kind.sigs.k8s.io/docs/user/quick-start/). 

It is maybe Kind is rightness.

### CODE REVIEW

**Comment by Gauravpadam** [comment]
For reference, PR #43250 was merged a while ago, Which accepted `kind` (all lowercase).

I leave it to the approvers to decide whether it should be `KinD`, `Kind` or `kind`.
My preference would incline with `KinD`

**Comment by sftim** [comment]
I checked https://kind.sigs.k8s.io/, which writes kind as “kind”.

/priority awaiting-more-evidence

File: content/en/blog/_posts/2020-05-21-wsl2-dockerdesktop-k8s.md
Comment (sftim) [inline content/en/blog/_posts/2020-05-21-wsl2-dockerdesktop-k8s.md:12]: `kind`, please.

**Review by sftim — CHANGES_REQUESTED** [review]
Please write kind as “kind”, all-lowercase.

**Comment by sftim** [comment]
/retitle Fix KinD typos in en docs

File: content/en/blog/_posts/2020-05-21-wsl2-dockerdesktop-k8s.md
Comment (yanggangtony) [reply content/en/blog/_posts/2020-05-21-wsl2-dockerdesktop-k8s.md]: Thanks for quick review.
Rebased.

**Comment by yanggangtony** [comment]
wait review, still a litter chang to go.

File: content/en/blog/_posts/2020-05-21-wsl2-dockerdesktop-k8s.md
Comment (tengqm) [reply content/en/blog/_posts/2020-05-21-wsl2-dockerdesktop-k8s.md]: Still not changed?

Comment (yanggangtony) [reply content/en/blog/_posts/2020-05-21-wsl2-dockerdesktop-k8s.md]: Rebased now.

File: content/en/blog/_posts/2020-05-21-wsl2-dockerdesktop-k8s.md
Comment (nate-double-u) [reply content/en/blog/_posts/2020-05-21-wsl2-dockerdesktop-k8s.md:1]: +1

**Comment by yanggangtony** [comment]
@sftim 
So your suggestion is like that .
>>For old blog articles, we should only change them if the authors agree (even for small fixups).
```
content/en/blog/_posts/2020-05-21-wsl2-dockerdesktop-k8s.md
content/en/blog/_posts/2021-06-21-writing-a-controller-for-pod-labels.md
content/en/blog/_posts/2021-12-09-pod-security-admission-beta.md
```
revert this changed.

Just changed the file 
```
content/en/blog/_posts/2020-08-26-kubernetes-release-1.19.md
content/en/docs/tutorials/security/ns-level-pss.md
```

File: content/en/blog/_posts/2020-05-21-wsl2-dockerdesktop-k8s.md
Comment (kumarankit999) [reply content/en/blog/_posts/2020-05-21-wsl2-dockerdesktop-k8s.md:1]: +1

**Comment by yanggangtony** [comment]
Thanks @sftim 
your suggest is right. Rebased. 
Now , we will not dirty the blogs without the agressment by its authors.

**Pull request merged**
